public_html/wp-content/plugins/wordcamp-organizer-survey/includes/admin-page.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Adds a page to the central admin.
+ */
+
+namespace WordCamp\OrganizerSurvey\AdminPage;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\OrganizerSurvey\{get_feature_id};
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\get_email_id as get_debrief_survey_email_id;
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\get_page_id as get_debrief_survey_page_id;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registered the appropriate filters and actions.
+ */
+function init() {
+	add_action( 'admin_menu', __NAMESPACE__ . '\admin_menu' );
+}
+
+/**
+ * Add a menu item.
+ */
+function admin_menu() {
+	add_menu_page(
+		__( 'WordCamp Organizer Survey', 'wordcamporg' ),
+		__( 'Organizer Survey', 'wordcamporg' ),
+		'manage_options',
+		get_feature_id(),
+		__NAMESPACE__ . '\render_menu_page',
+		'dashicons-feedback',
+		58
+	);
+}
+
+/**
+ * Debrief Survey: Get stats for each feedback for all sites.
+ *
+ * @return array[] An array of custom elements.
+ *                Each element has the following structure:
+ *                [
+ *                  'title' => string,
+ *                  'admin_url' => string,
+ *                  'email_url' => string,
+ *                  'responses_url' => string,
+ *                  'sent' => string,
+ *                  'responses' => string,
+ *                  'rate' => string,
+ *                ]
+ */
+function get_feedback_details() {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$wordcamps = get_posts( array(
+		'post_type' => WCPT_POST_TYPE_ID,
+		'post_status' => 'wcpt-closed',
+		'posts_per_page' => -1,
+	) );
+
+	$feedback_details = array();
+
+	foreach ( $wordcamps as $camp ) {
+		// It's a bit counter intuitive, but the site ID is the blog ID.
```

Comment:
This is a legacy inconsistency that stems from Core, where it was originally called a "blog", but then later called a "site". These days it's always called a site, but some of the old things like database schemas still say "blog" due to back-compat